docker/docker-compose-dev.yml

```diff
@@ -16,7 +16,7 @@ services:
 
   q_yaar_fast_worker:
     build: ./../q_yaar_platform
-    command: celery -A esamudaay_platform worker -E -l info --concurrency=8 --pool -n fast_worker@qyaar.com  -Q example
+    command: celery -A q_yaar worker -E -l info --concurrency=8 --pool=prefork -n fast_worker@qyaar.com  -Q example
```

Comment:
@paragjnath using prefork for now. Will move to gevent in future.

Move to gevent is a hassle and way too overkill for now. Let's use standard stuff instead of overengineering at this point.